utils/node_local_test.py

```diff
@@ -3082,6 +3082,40 @@ def _post(self, rc):
         if not self.aft.check_stderr:
             return
 
+        # Check stderr from a daos command.
+        # These should mostly be from the DH_PERROR_SYS or DH_PERROR_DER macros so check for
+        # this format.  There may be multiple lines and the two styles may be mixed.
+        if self.aft.check_daos_stderr:
+            stderr = self.stderr.decode('utf-8').rstrip()
+            for line in stderr.splitlines():
+
+                # This is what the go code uses.
+                if line.endswith(': DER_NOMEM(-1009): Out of memory'):
+                    continue
+
+                # This is what DH_PERROR_DER uses
+                if line.endswith(': Out of memory (-1009)'):
+                    continue
```

Comment:
Would it make sense to adjust DH_PERROR_DER to behave the same was as the GO code for consistency?

Reply:
I don't know how to change the go code ;)

In all seriousness though, DH_PERROR_DER is what Johann used initially and it's actually quite useful if the go and c are slightly different.

Reply:
> it's actually quite useful if the go and c are slightly different.

For testing purposes or real-world purposes? From a user's point of view, I would think consistent messages would be better.

Reply:
For testing.  The other thing with this file is the way it's written a lot of the error messages wrap in the file so searching for this is really tricky.  Using longer line length and putting these back together (as DH_PERROR is shorter this helps) makes it a lot easier to search for error messages.

There are also lots of places in the go code where you get a generic "daos failed" error message, and we should test for and update these at some point as well.

Once we've worked through these issues I'd have no problem with changing C and Go to use the same format though.